.circleci/config.yml

```diff
@@ -106,24 +106,17 @@ jobs:
           path: /tmp/dist
           destination: cli/new-production
 
-  e2e-cli-ng-snapshots:
+  e2e-cli-ivy:
```

Comment:
nit: why switch the order of these? makes the delta harder to review

Reply:
To keep similar e2e jobs (`e2e-cli` and `e2e-cli-ivy`) together.  

The old `e2e-cli-ng-snapshots` was after `e2e-cli-ivy-snapshots` because they shared the snapshot flag and only ran on some builds. Now `e2e-cli-ivy-snapshots` is instead `e2e-cli-ivy` and should instead be next to `e2e-cli`.

Reply:
Added.